pyclibrary/c_parser.py

```diff
@@ -31,6 +31,215 @@
 __all__ = ['win_defs', 'CParser']
 
 
+class Type(tuple):
```

Comment:
in the ```type_quals``` member of Type:
```python
x = Type('int', '*', type_quals=((), ('const',)))
```
means there is no typequal for 'int' and a 'const' typequal for the pointer (=you must not modify the pointer)